### PR TITLE
#49 added unit test for TkMeasured

### PR DESCRIPTION
**Pull request by d-molotchko**

### CODE REVIEW

**Comment by davvd** [comment]
Thanks for your pull request, let me find someone who can review it

**Comment by davvd** [comment]
@ggajos this pull request is for you,please review

File: src/test/java/org/takes/tk/TkMeasuredTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+     * Measured constant.
+     */
+    private static final String TEST = "test";
+
+    /**
+     * TkMeasured can create a response with HTTP header "X-Takes-Millis".
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void createMeasuredResponse() throws IOException {
```
Comment (ggajos) [inline src/test/java/org/takes/tk/TkMeasuredTest.java]: @d-molotchko let's name it "createsMeasuredResponse" (http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names)

File: src/test/java/org/takes/tk/TkMeasuredTest.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            Matchers.containsString(tkMeasuredHeader)
+        );
+    }
+
+    /**
+     * TkMeasured can create a response with custom HTTP header.
+     * @throws IOException If some problem occurs
+     */
+    @Test
+    public void createMeasuredResponseWithCustomHeader() throws IOException {
```
Comment (ggajos) [inline src/test/java/org/takes/tk/TkMeasuredTest.java]: @d-molotchko let's name it "createsMeasuredResponseWithCustomHeader" (http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names)

File: src/test/java/org/takes/tk/TkMeasuredTest.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+ * @author Dmitry Molotchko (dima.molotchko@gmail.com)
+ * @version $Id$
+ * @since 0.9.1
+ */
+public final class TkMeasuredTest {
+
+    /**
+     * Measured constant.
+     */
+    private static final String TEST = "test";
```
Comment (ggajos) [inline src/test/java/org/takes/tk/TkMeasuredTest.java]: @d-molotchko let's avoid coupling tests together like this, we can use different string in each test, no?

Comment (d-molotchko) [inline src/test/java/org/takes/tk/TkMeasuredTest.java]: Even if they are the same? This string used as measured string in both tests, but there are other different strings - HTTP response header. Maybe its name is not clean. I can rename it to MEASURED_STRING

Comment (d-molotchko) [inline src/test/java/org/takes/tk/TkMeasuredTest.java]: @ggajos I have read whole article http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html I agree with using different strings in tests, so I'll remake it

**Comment by ggajos** [comment]
@rultor merge please

File: src/test/java/org/takes/tk/TkMeasuredTest.java [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+ */
+public final class TkMeasuredTest {
+
+    /**
+     * TkMeasured can create a response with HTTP header "X-Takes-Millis".
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void createsMeasuredResponse() throws IOException {
+        final String tkMeasuredHeader = "X-Takes-Millis";
```
Comment (yegor256) [inline src/test/java/org/takes/tk/TkMeasuredTest.java]: see http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

File: src/test/java/org/takes/tk/TkMeasuredTest.java [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        );
+    }
+
+    /**
+     * TkMeasured can create a response with custom HTTP header.
+     * @throws IOException If some problem occurs
+     */
+    @Test
+    public void createsMeasuredResponseWithCustomHeader() throws IOException {
+        final String customHeader = "X-Custom-Takes-Millis";
```
Comment (yegor256) [inline src/test/java/org/takes/tk/TkMeasuredTest.java]: same: http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

**Comment by yegor256** [comment]
@d-molotchko two comments from me above

File: src/test/java/org/takes/tk/TkMeasuredTest.java (hunk 4 above)
Comment (d-molotchko) [inline src/test/java/org/takes/tk/TkMeasuredTest.java]: I suppose, it must be just header?

File: src/test/java/org/takes/tk/TkMeasuredTest.java (hunk 5 above)
Comment (d-molotchko) [inline src/test/java/org/takes/tk/TkMeasuredTest.java]: same as above?

Comment (yegor256) [inline src/test/java/org/takes/tk/TkMeasuredTest.java]: yes, just `header` is enough. there are no more "headers" in the scope, no point to give it a "last name". just first name is enough :)

File: src/test/java/org/takes/tk/TkMeasuredTest.java [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+import org.takes.rs.RsPrint;
+
+/**
+ * Test case for {@link TkMeasured}.
+ * @author Dmitry Molotchko (dima.molotchko@gmail.com)
+ * @version $Id$
+ * @since 0.9.1
```
Comment (yegor256) [inline src/test/java/org/takes/tk/TkMeasuredTest.java]: @d-molotchko also, this is wrong. the next version is 0.10

File: src/test/java/org/takes/tk/TkMeasuredTest.java (hunk 5 above)
Comment (d-molotchko) [inline src/test/java/org/takes/tk/TkMeasuredTest.java]: I see. I'll do.

**Comment by d-molotchko** [comment]
@yegor256 review this again, please

**Comment by ggajos** [comment]
@yegor256 I think I should be assigned to this PR, no?

**Comment by ggajos** [comment]
@davvd I think I should be assigned to this PR, no?

**Comment by yegor256** [comment]
@rultor merge it

**Comment by yegor256** [comment]
@ggajos this "assignment" mechanism doesn't work with projects that belong to individual (not org) accounts, unfortunately

**Comment by d-molotchko** [comment]
@yegor256  build failed, what should I do?

**Comment by yegor256** [comment]
@d-molotchko ask me to ask rultor to try again

**Comment by yegor256** [comment]
@rultor try to merge again

**Comment by davvd** [comment]
@ggajos Many thanks! **18 mins** were added to your account in Transaction ID `AP-8J624854JX3536507`; you're getting extra minutes here (c=3); +18 added to your rating, at the moment it is: [+3112](http://www.netbout.com/b/35094?open=rating)

**Comment by davvd** [comment]
@rultor please deploy

**Pull request merged**